src/main/java/reposense/system/CustomLogFormatter.java

```diff
@@ -12,11 +13,17 @@
 public class CustomLogFormatter extends SimpleFormatter {
 
     private static final DateFormat dateFormat = new SimpleDateFormat("hh:mm:ss");
+    private static final String ERROR_HIGHLIGHT = "\u001b[37;41m[ERROR]\u001b[0m";
 
     @Override
     public synchronized String format(LogRecord record) {
         StringBuilder builder = new StringBuilder();
         builder.append(dateFormat.format(new Date(record.getMillis()))).append(" - ");
+
+        if (record.getLevel().intValue() >= Level.WARNING.intValue()) {
```

Comment:
You are appending `[ERROR]` to severity above `warning` right?


Your PR message:

>It is difficult to distinguish `Level.WARNING` and `Level.SEVERE`
>...
>Let's make these messages more distinct by inserting a highlighted
>`[ERROR]` status before the message.

Not sure it would help in readability/distinguishability.

Reply:
Would specifying `[WARNING]` (in another colour perhaps) instead for warnings be more desirable?